src/mapgen/treegen.cpp

```diff
@@ -32,6 +32,16 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 namespace treegen
 {
 
+void TreeDef::resolveNodeNames()
+{
+	getIdFromNrBacklog(&trunknode.param0, "", CONTENT_UNKNOWN);
```

Comment:
i don't see `CONTENT_UNKNOWN` being used in this role in the engine at all.
not even sure what happens when you manually place this.